### DISCUSSION TITLE
PWA

### DISCUSSION THREAD

**Discussion by OfficialCRUGG** [modrinth/.github#142] [Feature requests]
Quite simple: Make the Modrinth Site a PWA.
Would be especially useful on mobile devices.

**Comment by triphora** [comment]
I don't think this would really be necessary—it would be a lot of work to implement and maintain for very minimal benefit, especially considering we have a desktop app already

**Comment by OfficialCRUGG** [comment]
What effort exactly? For simple PWA functionality, all that is needed would be a manifest json file and some logo assets. 
Would allow mobile users to have a nicer experience when adding the app to their home screen.

**Comment by triphora** [comment]
The idea was explored in knossos#1102 but in order to get "true" support we would need to rewrite the notifications system. Otherwise the result would be half-assed, and we'd rather not release something that isn't fully functional.

**Comment by OfficialCRUGG** [comment]
Push Notifications and PWA are in no relation though. You don't need to have a PWA for Push Notifications or vice versa. Many sites are available as PWA, but don't have push notifications. In fact, iOS for example doesn't even support push notifications for websites/PWAs.
I personally don't really see what speaks against just adding it. A basic PWA setup would be a one-time minimal effort, not break anything existing, and would simply let users add the app to their home screen with a fancy icon and no address bar (without the PWA setup, it'd just be a shortcut)
